wdqs/0.3.0/.travis/build-deploy.sh

```diff
@@ -6,5 +6,4 @@ docker build --pull "$DIR/../" -t wikibase/wdqs:0.3.0 -t wikibase/wdqs:latest
 
```

Comment:
The latest shouldn't still be here

Reply:
ack!

I think this indicates the complexity of the CI for pushing and tagging these!

Reply:
yes, and also that there is so much noise in the commit with the duplicated files.

IMHO we should look at using a build context that is higher and stop duplicating them everywhere see: https://phabricator.wikimedia.org/T194399

Reply:
yup